src/components/Team/RecruitmentApplyPage/Steps/BasicInfoStep/index.tsx

```
@@ -0,0 +1,368 @@
+import XIcon from 'assets/svg/Team/x-icon.svg';
+import DeptSelect from 'components/Team/ProfilePage/components/DeptSelect';
+import FormField from 'components/Team/ProfilePage/components/FormField';
+import StepIndicator from 'components/Team/ProfilePage/components/StepIndicator';
+import ActivityHistoryModal from 'components/Team/RecruitmentApplyPage/components/ActivityHistoryModal';
+import SkillTagInput from 'components/Team/RecruitmentApplyPage/components/SkillTagInput';
+import {
+  APPLY_INTRODUCTION_MAX_LENGTH,
+  APPLY_NICKNAME_MAX_LENGTH,
+} from 'components/Team/RecruitmentApplyPage/schema';
+import { APPLY_STEPS } from 'components/Team/RecruitmentApplyPage/types';
+import { Controller, useFieldArray, useFormContext, useFormState, useWatch } from 'react-hook-form';
```

ESLint

[error] `react-hook-form` import should occur before import of `api/auth`

(import/order)